frontend/public/components/utils/resource-log.tsx

```diff
@@ -191,8 +193,16 @@ export const LogControls: React.FC<LogControlsProps> = ({
       <div className="co-toolbar__group co-toolbar__group--left">
         <div className="co-toolbar__item">{showStatus()}</div>
         {dropdown && <div className="co-toolbar__item">{dropdown}</div>}
-
         <div className="co-toolbar__item">{logTypeSelect(!hasPreviousLog)}</div>
+        <Link
```

Comment:
Do we need to hide this if the pod is not in crashloop?  Any pod can be debugged no matter its state.

Reply:
^ @megan-hall, do you have any input on this?

Reply:
This is what I had mocked up in the design doc for this use case: https://docs.google.com/document/d/1b2n9Ox4xDNJ6AkVsQkXc5HyG8DXJIzU8tF6IsJCiowo/edit#heading=h.kqzmgowlw12d

<img width="833" alt="Screen Shot 2021-07-22 at 11 26 24 AM" src="https://user-images.githubusercontent.com/62031580/126665516-069ef371-ca7d-4bba-a7a6-48d071348e74.png">

Reply:
The conditions for the debug link in 3.x can be seen at https://github.com/openshift/origin-web-console/blob/c37982397087036321312172282e139da378eff2/app/scripts/directives/resources.js#L33-L53

Reply:
@spadgett, can you confirm the 3.x conditions above are correct?

Reply:
The 3.x logic looks good to me. We'd need to adjust the tooltip message, though.